src/test/starknet.t.sol

```diff
@@ -83,6 +89,11 @@ interface DaiLike {
 contract StarknetTests is DssSpellTestBase, ConfigStarknet {
 
     function testStarknet() public {
+        if (l2Spell != 0) {
+            // Ensure the Pause Proxy has some ETH for the Starknet Spell
+            assertGt(pauseProxy.balance, 0);
```

Comment:
Is there a sensible minimum we could use here instead? Seems like pp could be gt 0 but lt what's needed.

Reply:
perhaps `0.05 ethers`?  L2 transactions take so little gas, I think the L2 teams would need to suggest a minimum.